.allow_skipping_tests

```diff
@@ -19,6 +22,8 @@ helpers/template_helper.rb
 jobs/application_job.rb
 mailers/application_mailer.rb
 models/application_record.rb
+models/case_group.rb
+models/case_group_membership.rb
```

Comment:
Would you mind adding these spec files, and just doing a "has a valid factory" test? People are more encourage to write tests if the spec file exists!